lib/screens/cash_home/cash_home.dart

```diff
@@ -17,7 +17,10 @@ void showAlert(BuildContext context) {
   showDialog(child: new ContactsConfirmationScreen(), context: context);
 }
 
-void onChange(CashWalletViewModel viewModel, BuildContext context) async {
+void onChange(CashWalletViewModel viewModel, BuildContext context, {bool initial = false}) async {
+  if (initial) {
+    viewModel.syncContacts([]);
```

Comment:
that's for fixing some issue with contact not appearing?

Reply:
Yes